packages/content/src/relationships/ArbitraryRelationshipCreationContent.ts

```diff
@@ -0,0 +1,26 @@
+import { ISerializable, Serializable, serialize, type, validate } from "@js-soft/ts-serval";
+import { ContentJSON } from "../ContentJSON";
+
+export interface ArbitraryRelationshipCreationContentJSON extends ContentJSON {
+    "@type": "ArbitraryRelationshipCreationContent";
+    value: any;
+}
+
+export interface IArbitraryRelationshipCreationContent extends ISerializable {
+    value: any;
+}
+
+@type("ArbitraryRelationshipCreationContent")
+export class ArbitraryRelationshipCreationContent extends Serializable implements IArbitraryRelationshipCreationContent {
+    @serialize()
```

Comment:
@stnmtz JSONWrapper uses `@serialize({ any: true })` for any contents.

1. what's the purpose of that?
2. should we use that for these `Arbitrary*` classes too?

Reply:
@stnmtz wrote

> Ich glaub wir müssen any benutzen, ansonsten meckert er, wenn es z.B. undefined ist - und weil er keinen Datentyp bekommt, weiß er nicht genau wie er das zu validieren hat. Aber nur eine Vermutung

Reply:
also IMO it doesn't hurt to just use it.

Reply:
erm @Magnus-Kuhn.. why did you remove the unknown keyword?

Reply:
I thought we were doing things JSONWrapper-style...

Reply:
nope. we use `@serialize({ any: true })` to disable some ts-serval validation and serialization logic, but we still want the compiler to teat it as arbitrary content.

Additionally: JSONWrapper should also use unknown instead of any, back then we didn't knew about this.